care/facility/api/serializers/notification.py

```diff
@@ -1,11 +1,32 @@
+from django.contrib.auth import get_user_model
 from rest_framework import serializers
 
 from care.facility.models.notification import Notification
 from care.users.api.serializers.user import UserBaseMinimumSerializer
 from config.serializers import ChoiceField
 
+User = get_user_model()
 
-class NotificationSerializer(serializers.ModelSerializer):
+
+class BareMinimumCausedBySerializer(serializers.ModelSerializer):
+    id = serializers.UUIDField(source="external_id", read_only=True)
+
+    class Meta:
+        model = User
+        fields = ("id", "first_name", "last_name", "user_type")
```

Comment:
We can use the `UserBaseMinimumSerializer` instead of this

Reply:
@sainak `BareMinimumCausedBySerializer` has very less data compared to `UserBaseMinimumSerializer`, should I still use `UserBaseMinimumSerializer`

Reply:
yes, adding new serializer for each view will cause duplication, also place serialzers in their respective module